test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,71 @@
+./test_scripts/API/SetAudioStreamingIndicator/053_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/054_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/055_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_navi_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/056_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_comm_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/057_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_comm_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/058_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_comm_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/059_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_comm_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/060_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_media_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_non-media_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/062_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_navi_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/063_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_comm_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_missing_:
+./test_scripts/API/SetAudioStreamingIndicator/065_ATF_N_wrong_Json_wrong_serviceType_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_serviceType_missing_wrong_wrongtype
```

Comment:
@istoimenova I did not found such requirements for this CRQ, please help to find. I think this check of protocol level and if serviceType will be missing SDL will reject this message as malformed message.